templates/blank/{{cookiecutter.project_slug}}/mkdocs.yaml

```diff
@@ -0,0 +1,64 @@
+site_name: {{ cookiecutter.project_name }}
+repo_url: https://git.int.avast.com/{{ cookiecutter.project_repo_name }}
```

Comment:
I guess this should be changed so we're not mixing the internal and public git servers?